package.json

```diff
@@ -21,20 +21,31 @@
   },
   "bugs": "https://github.com/helmetjs/ienoopen/issues",
   "scripts": {
-    "pretest": "standard --fix",
-    "test": "mocha"
+    "pretest": "npm run lint",
+    "prepublishOnly": "npm run build",
+    "lint": "tslint --project . --fix",
+    "test": "jest --config test/jest-config.json",
+    "build": "tsc"
   },
+  "main": "./dist/index.js",
+  "typings": "./dist/index.d.ts",
```

Comment:
Is this correct? Is there more I need to do here?

Reply:
That's right. For larger projects, assuming that all your module structure is emitted into `dist` correctly, the imports in the index.d.ts file will correctly look up other modules that it depends on.